ivy/src/main/scala/sbt/ivyint/CachedResolutionResolveEngine.scala

```diff
@@ -82,17 +91,19 @@ private[sbt] class CachedResolutionResolveCache() {
       def nextConfMap(dd: DependencyDescriptor, previous: Map[String, Array[String]]): Map[String, Array[String]] =
         previous map {
           case (rootModuleConf, vs) =>
-            rootModuleConf -> (vs flatMap { conf =>
+            rootModuleConf -> (vs flatMap { conf0 =>
+              val conf = if (conf0 == "default(compile)") "compile"
```

Comment:
This is a hack on traversing Ivy modules across different configurations. `"default(compile)"` is treated specially because it happens to show up often.